python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3371,8 +3371,8 @@ def test_series_astype_null_cases():
 
     # string to other
     assert_eq(
-        gd.Series([1, 2, None, 3], dtype="int32"),
-        gd.Series(["1", "2", None, "3"]).astype("int32"),
+        gd.Series([1, 2, 3], dtype="int32"),
```

Comment:
May be need to move to other test-case body as this test case is meant for null cases.

Reply:
Removed this test as it is no more relevant. Instead testing with and without null separately in the newly added test.